src/compute-types/src/plan/mod.rs

```diff
@@ -850,9 +850,10 @@ impl<T: timely::progress::Timestamp> Plan<T> {
     )]
     fn lower_dataflow(
         desc: DataflowDescription<OptimizedMirRelationExpr>,
-        enable_reduce_mfp_fusion: bool,
+        features: &OptimizerFeatures,
     ) -> Result<DataflowDescription<Self>, String> {
-        let context = lowering::Context::new(desc.debug_name.clone(), enable_reduce_mfp_fusion);
+        let context =
+            lowering::Context::new(desc.debug_name.clone(), features.enable_reduce_mfp_fusion);
```

Comment:
If we think that we might add more flags that influence MIR-LIR lowering, we can also consider giving the entire `OptimizerFeatures` to the context. But fine to only go there once we add the second relevant flag.

Reply:
Ah, yes. I forgot to change this.